kong/runloop/handler.lua

```diff
@@ -1145,6 +1145,7 @@ return {
       if not match_t then
         -- tracing
         if span then
+          span:set_attribute("http.status_code", 404)
```

Comment:
this line is adding the attribute to the `router` span, which is not always enabled, I think we should add it to the root (`kong`) span as well, which is the span that normally contains this information, see [here](https://github.com/Kong/kong/blob/6b3d3c5a49f89e20f277136c1df2109e6e6e1df1/kong/tracing/instrumentation.lua#L323).

Also, is [this](https://github.com/Kong/kong/pull/11406) the same (or a very similar) issue?

Reply:
Ahh, Yeah, thanks! Being an attribute of root span  is much more reasonable, and it seem optional to be added to router span, and I'd like just add it to root span.

same purpose of recording the status as #11406 , but it can't cover the issue in this case.

Reply:
@liverpool8056 unless I'm missing something, I think https://github.com/Kong/kong/pull/11406 solves this problem too. I just ran your test without **this** fix in `runloop` and it's passing.